projects/ngx-d3timeline/src/lib/selector/memoized-selector.ts

```diff
@@ -12,7 +12,9 @@ export class MemoizedSelector {
   ) {}
 
   execute(state: State) {
-    const args = this.inputSelectors.map(selector => selector.execute(state));
+    const args =
+      this.inputSelectors &&
```

Comment:
> Why is this line necessary? Is it because of the selector factories? Should we be using a different create function for them? Or not due to multiple nesting levels?

Its not necessary I've removed it